tasks/pipeline-pipdeps-multi-venvs.yml

```diff
@@ -0,0 +1,85 @@
+---
+###########################################################
+#   python dependencies (pip packages) with separate virtualenvs
+###########################################################
+
+- name: "Create virtualenv for archivematica-mcp-server"
+  pip:
+    name: "pip"
+    chdir: "{{ archivematica_src_dir }}/archivematica/src/MCPServer"
+    virtualenv: "/usr/share/python/archivematica-mcp-server"
+    state: "latest"
+
+- name: "Create virtualenv for archivematica-mcp-client"
+  pip:
+    name: "pip"
+    chdir: "{{ archivematica_src_dir }}/archivematica/src/MCPClient"
+    virtualenv: "/usr/share/python/archivematica-mcp-client"
+    state: "latest"
+
+- name: "Create virtualenv for archivematica-dashboard"
+  pip:
+    name: "pip"
+    chdir: "{{ archivematica_src_dir }}/archivematica/src/dashboard"
+    virtualenv: "/usr/share/python/archivematica-dashboard"
+    state: "latest"
+
+
+- name: "Install common pip reqs in virtualenv archivematica-mcp-server"
+  pip:
+    chdir: "{{ archivematica_src_dir }}/archivematica/src/archivematicaCommon"
+    requirements: "{{ 'requirements/test.txt' if is_dev else 'requirements.txt'}}"
```

Comment:
Happy to see it installing the testing reqs if is_dev is set - I was going to implement this for AM but found it here first.